src/insert/spacefinder/article.ts

```diff
@@ -462,17 +498,21 @@ const init = async (fillAdSlot: FillAdSlot): Promise<boolean> => {
 		return Promise.resolve(false);
 	}
 
-	await addInlineAds(fillAdSlot);
+	// We add the first inline ad before finding a place for an im so that the inline1 doesn't get pushed down
+	await addFirstInlineAd(fillAdSlot);
 
 	const im = window.guardian.config.page.hasInlineMerchandise
 		? attemptToAddInlineMerchAd(fillAdSlot)
 		: Promise.resolve(false);
 	const inlineMerchAdded = await im;
 	if (inlineMerchAdded) await waitForAdvert('dfp-ad--im');
```

Comment:
do we still need `waitForAdvert` here? we don't wait for first inline, so should be fine to remove?

Reply:
Do we know what this function is used for? I've traced it back and it looks like a pretty old bit of code that was split out from some dfp functionality: https://github.com/guardian/frontend/blob/e0013c355da31e7902a7e985a3d46c861729469a/static/src/javascripts/projects/commercial/modules/dfp/wait-for-advert.js

This is the only time we call function so if we think we don't need it we could delete the file too?

Reply:
I don't think we need to return the im promise anymore either, could it simplify to:
```ts
if (window.guardian.config.page.hasInlineMerchandise) {
		await attemptToAddInlineMerchAd(fillAdSlot);
}